codemon/CodemonListen.py

```diff
@@ -38,7 +38,10 @@ def isModified(event):
   if filename != foldername and filename != "prog" and filename != "input.txt": 
     print(colored.yellow('\nChange made at '+ filename))
     print(colored.cyan('\nCompiling '+ filename))
-    if os.system('g++ ' + filename + ' -o ' + 'prog') == 0:
+    folder,lang = filename.split('.')

```

Comment:
rename `folder` to `dirName`

Reply:
Rename `lang` to `fileExtension`. Also, think about how could we use the extension to decide which compilation command is to be used, since we will be supporting C++, python & java for codemon.